docs/source-1.0/spec/aws/aws-json.rst.template

```diff
@@ -192,6 +192,45 @@ In ``awsJson1_1``, servers SHOULD only send the error's
 for both protocols. See `Operation error serialization`_ for full details on how to
 deserialize errors for |quoted shape name|.
 
+.. smithy-trait:: aws.protocols#awsQueryError
+.. _aws.protocols#awsQueryError-trait:
+
+------------------------------------------
+``aws.protocols#awsQueryCompatible`` trait
+------------------------------------------
+
+Summary
+    When using ``awsQuery`` protocol, ```awsQueryError`` trait allows a custom code value defined for an error response.
```

Comment:
You might want to double check with someone who's native speaker, but I think that this would read better as


>    When using ``awsQuery`` protocol, ``awsQueryError`` trait allows a custom code value *to be* defined for an error response.

Also, you have three backticks instead of two before `awsQueryError`.